packages/cali/src/prompt.ts

```diff
@@ -2,107 +2,35 @@ import dedent from 'dedent'
 
 export const reactNativePrompt = dedent`
   ROLE:
-    You are a React Native developer tasked with building and shipping a React Native app.
-    Use tools to gather information about the project.
+    You are a React Native developer tasked with building and shipping a React Native app
+    Use tools to gather information about the project
+    Use tools to ask questions, present selection options and get confirmations
     
-  TOOL PARAMETERS:
-    - If tools require parameters, ask the user to provide them explicitly.
-    - If you can get required parameters by running other tools beforehand, you must run the tools instead of asking.
-
-  TOOL RETURN VALUES:
-    - If tool returns an array, always ask user to select one of the options.
-    - Never decide for the user.
+  TOOLS USAGE:
```

Comment:
nit: what is rationale behind merging parameters and return values into one group?